certified-connectors/FactSet/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1417 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "FactSet",
+    "description": "Prepare for client meetings using the FactSet connector to access key company information in a custom branded, presentation ready PDF. Stay on top of relevant news and events for companies of interest and highlight important topics to drive your discussion with FactSet Signals. Alternatively, use Digital Cards to collaborate with colleagues and send summaries of actionable insights via Teams.",
+    "contact":{
+      "name" : "FactSet Workflow Automation Support",
+      "url": "https://www.factset.com/",
+      "email":"WorkflowAutomation@factset.com"
+    },
+    "version": "1.0"
+  },
+  "x-ms-connector-metadata":[
+    {
+      "propertyName": "Website",
+      "propertyValue": "https://www.factset.com"
+    },
+    {
+      "propertyName": "Privacy policy",
+      "propertyValue": "https://www.factset.com/legal"
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Finance;Data"
+    }
+  ],
+  "host": "api.factset.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/book-builder-api/book-list": {},
+    "/book-builder-api/template-list": {},
+    "/book-builder-api/get-book-id/{ticker}": {},
+    "/signals/v2/events/headlines": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "requestId": {
+                        "type": "string",
+                        "description": "requestId"
+                      },
+                      "signalId": {
+                        "type": "string",
+                        "description": "signalId"
+                      },
+                      "signalName": {
+                        "type": "string",
+                        "description": "signalName"

```

Comment:
@krithsai All the description values of properties in the swagger are expected to be free from grammatical and spell errors. Please provide valid description values.
Docs - [Define operation and parameter summaries and descriptions](https://docs.microsoft.com/connectors/custom-connectors/certification-submission#define-operation-and-parameter-summaries-and-descriptions)